crowdfunding_compassion/models/website.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    def rule_is_enumerable(self, rule):
+        """ Create a route parameter named no_sitemap. Set it
+            to true to exclude this route from the sitemap
+            generation.
+            :type rule: werkzeug.routing.Rule
+            :rtype: bool
+        """
+        if rule.endpoint.routing.get('no_sitemap'):
+            return False
+        return super(Website, self).rule_is_enumerable(rule)
```

Comment:
Can you move this file inside module `compassion-modules/cms_form_compassion`? Like that we will have the attribute available for all our website modules and not only the crowdfunding.